news/_posts/2022-10-30-playtest-20221030.md

```diff
@@ -0,0 +1,89 @@
+---
+kind: article
+title: "Playtest 20221030"
+author: "PunkPun, Therapist"
+created_at: 2022-10-30 22:00 +0000
+disqus_id: "playtest-20221030"
+permalink: "/news/playtest-20221030/"
+---
+
+Welcome back Commanders! After more than a year and a half the OpenRA team proudly presents [Playtest 20221030]({{ '/download' | relative_url }}). It's finally here!!
+
+There's been a ton of refactors under the hood -- it took us a very long time to catch and squash all those pesky bugs -- and we're excited to finally showcase a number of new features that've been built up over the years!
+
+One of the highlight features is the new hierarchical pathfinder. It should increase performance massively on large or more complex maps.
+
+<figure>
+  <img src="{{ '/images/news/20221031-hpf.png' | relative_url }}" style="width: 600px" alt="HPF helps units better predict obstacles ahead of them" />
+  <figcaption>HPF helps units better predict obstacles ahead of them.</figcaption>
+</figure>
+
+Our netcode has had some major improvements in both low and high latency games.
+
+<figure>
+  <video width="700" autoplay loop muted>
+    <source src="{{ '/images/news/20221031-td.mp4' | relative_url }}" type="video/mp4">
+    <source src="{{ '/images/news/20221031-td.ogv' | relative_url }}" type="video/ogg">
+    <img src="{{ '/images/news/20221031-td.png' | relative_url }}" width="700" />
+  </video>
+  <figcaption>Forces collide</figcaption>
+</figure>
+
+We've added transient text notifications matching speech notifications. Now you can play without sound!
```

Comment:
```suggestion
We added transient text notifications to match speech notifications. Now you can play without sound!
```

We could include something here along the lines of "Now it is harder than ever to miss important notifications during battle" to give this a spin that it doesn't just benefit people who play without sound. (nts: revisit)

Reply:
Perhaps